lib/deploy-endpoint.js

```diff
@@ -22,11 +25,13 @@ function serviceFindOrCreate(client, serviceName, clusterSize, callback) {
     if (err) {
       debug('client.serviceFindOrCreate(%s, 1): %s', serviceName, err);
       if (err.statusCode === 401) {
-        console.error(
-          'Cannot access remote. If authentication is required,' +
-          ' credentials should be given in the URL.');
+        glb
+        .error('Cannot access remote.  If authentication is required, ' +
```

Comment:
extra SPC before "If", and its easier to compare diffs if other whitespace is the same. reindent to be the same as the previous code, all you needed to do was replace console with glb, I think.